themes/haystack/layouts/_default/tutorials.html

```diff
@@ -1,120 +1,177 @@
-              fill-rule="evenodd"
-              clip-rule="evenodd"
-            />
-          </svg>
-          {{ .Params.contribute.text }}
-        </a>
-      </section>
-
-      {{/* Tutorial accordions */}}
-      <div class="accordions">
-        {{/* Beginner */}}
-        <details class="accordion-js" open>
-          <summary class="accordion-title">
-            Beginner
-            <svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
-              <path
-                fill="#fefefd"
-                d="M98,190.06,237.78,353.18a24,24,0,0,0,36.44,0L414,190.06c13.34-15.57,2.28-39.62-18.22-39.62H116.18C95.68,150.44,84.62,174.49,98,190.06Z"
-              />
-            </svg>
-          </summary>
-          <div class="content">
-            {{ range where (where .Pages "Section" "tutorials")  ".Params.level" "beginner" }}
-              <a
-                class="tutorial-card category-{{ lower .Params.category }}"
-                href="{{ .RelPermalink }}"
-              >
-                <h3>{{ .Params.title }}</h3>
-                <p>{{ .Params.Description }}</p>
-              </a>
-            {{ end }}
+            <div>
+              {{/* Sorting */}}
+              {{/*  <div class="select-wrapper">
+                <select id="sort-select" onchange="filterAndSort();">
+                  <option value="new">Sort by:</option>
+                </select>
+              </div>  */}}
+            </div>
           </div>
-        </details>
-
-        {{/* Intermediate */}}
-        <details class="accordion-js">
-          <summary class="accordion-title">
-            Intermediate
-            <svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
-              <path
-                fill="#fefefd"
-                d="M98,190.06,237.78,353.18a24,24,0,0,0,36.44,0L414,190.06c13.34-15.57,2.28-39.62-18.22-39.62H116.18C95.68,150.44,84.62,174.49,98,190.06Z"
-              />
-            </svg>
-          </summary>
-          <div class="content">
-            {{ range where (where .Pages "Section" "tutorials")  ".Params.level" "intermediate" }}
-              <a
-                class="tutorial-card category-{{ lower .Params.category }}"
-                href="{{ .RelPermalink }}"
-              >
-                <h3>{{ .Params.title }}</h3>
-                <p>{{ .Params.Description }}</p>
-              </a>
+
+          {{/* Tutorials grid */}}
+          <div class="tutorial-grid">
+            {{ range where (.Site.Pages.ByParam "new") "Layout" "tutorial" }}
```

Comment:
I believe this order should be reversed. For me, "new" content always comes last.

Reply:
@bilgeyucel Not sure what you mean, atm the new tutorials are listed on the top like in the figma design, do you want them listed at the bottom instead?
![new](https://user-images.githubusercontent.com/53526597/218053030-37497e84-a3b6-437f-b509-9b9652b06c04.png)

Reply:
@carlosgauci  No, new tagged tutorials should come first. 
The problem is they are ordered according to the `new` attribute, and the possible values are `true` or `false`. Since `false` comes before `true` in the string order (because the first letter F comes before T), `new` tagged tutorials come as last on the overview page (at least this is my prediction)
I thought the easiest way would be to reverse the order but I am open to change the value of `new` attribute if you have suggestions

Reply:
@bilgeyucel ah I see, you're right, it was acting differently if you just added `new: true` and left the others without the tag.

Updated it so it's more consistent, news tutorials will show on top and all the others after regardless if they're tagged or not. Also we can change True/False to capital in the frontmatter to match the other (toc)